src/toil/common.py

```diff
@@ -23,10 +23,15 @@
 import time
 import uuid
 import warnings
-from argparse import (ArgumentDefaultsHelpFormatter,
+
+from ruamel.yaml import YAML
+from ruamel.yaml.comments import CommentedMap
+from configargparse import ArgParser, YAMLConfigFileParser
+from argparse import (SUPPRESS,
+                      ArgumentDefaultsHelpFormatter,
                       ArgumentParser,
                       Namespace,
-                      _ArgumentGroup)
+                      _ArgumentGroup, _StoreAction, _AppendAction, _StoreFalseAction, _StoreTrueAction)
```

Comment:
Do these get used?

Reply:
Some of the custom action classes and the config file generator use these classes. Ideally I'd like for them to only use argparse.Action, but I haven't found a workaround for that.